app/scripts/modules/core/pipeline/config/triggers/docker/dockerTriggerOptions.directive.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.pipeline.config.triggers.docker.options.directive', [
+    require('../../../../../core/config/settings.js'),
+  ])
+  .directive('dockerTriggerOptions', function () {
+    return {
+      restrict: 'E',
+      templateUrl: require('./dockerTriggerOptions.directive.html'),
+      bindToController: {
+        command: '=',
+      },
+      controller: 'dockerTriggerOptionsCtrl',
+      controllerAs: 'vm',
+      scope: {}
+    };
+  })
+  .controller('dockerTriggerOptionsCtrl', function ($scope, Restangular) {
+    // These fields will be added to the trigger when the form is submitted
+    this.command.extraFields = {};
+
+    this.viewState = {
+      tagsLoading: false,
+      loadError: false,
+      selectedtag: null,
```

Comment:
Case is off here - should be `selectedTag` (same with L35,36,62, and 12/13 in template)